x/gravity/keeper/keeper.go

```diff
@@ -55,7 +57,7 @@ func NewKeeper(
 	stakingKeeper types.StakingKeeper,
 	bankKeeper types.BankKeeper,
 	slashingKeeper types.SlashingKeeper,
-	distributionKeeper types.DistributionKeeper,
+	distributionKeeper distrkeeper.Keeper,
```

Comment:
Why this change? The type should be from gravity module.

Reply:
cz we were unable to import get or set functions when we tried to use them ...so to call feepool.get/set functions we had to make this change

Reply:
@byte-bandit

Reply:
> cz we were unable to import get or set functions when we tried to use them ...so to call feepool.get/set functions we had to make this change

I see, distribution keeper has SDK collection fields.

Reply:
Yupp looks good.